packages/jest-core/src/lib/createContext.ts

```diff
@@ -7,12 +7,12 @@
 
 import type {TestContext} from '@jest/test-result';
 import type {Config} from '@jest/types';
-import type {HasteMapObject} from 'jest-haste-map';
+import type {IHasteFS, IModuleMap} from 'jest-haste-map';
 import Runtime from 'jest-runtime';
 
 export default function createContext(
   config: Config.ProjectConfig,
-  {hasteFS, moduleMap}: HasteMapObject,
+  {hasteFS, moduleMap}: {hasteFS: IHasteFS; moduleMap: IModuleMap},
```

Comment:
why do we have an inline type here instead of using a named one?

Reply:
I mostly meant, why is this `HasteMapObject` thing not exported from `jest-haste-map`?